src/styles/components/_home.scss

```diff
@@ -63,6 +63,7 @@ $breakpoint-top-section-scale-up: 70rem;
       margin: 1.25rem 0 0; // top: 20px/16; bottom&left&right: 0
       line-height: 1.75rem; // 28px/16
       letter-spacing: 0.02em;
+      max-width: 70ch;
```

Comment:
Oh great idea here! 👏

Reply:
I have Dustin to thank for that! :)

Reply:
I just removed that stray classname too. Thanks! :)